components/src/core/components/CardTable/Cell/cell-mixin.ts

```diff
@@ -3,7 +3,7 @@ import {decoratorMixin} from '../Decorator/decorator-mixin';
 
 export const cellMixin = defineComponent({
   mixins: [decoratorMixin],
-  inject: ['screenState'],
+  inject: ['tableProps', 'screenState'],
```

Comment:
`tableProps` are already injected via [decorator mixin](https://github.com/orangehrm/oxd/blob/ent/components/src/core/components/CardTable/Decorator/decorator-mixin.ts#L4).

Reply:
Removed 'tableProps' and  'screenState' as both are available in decorator mixin